db/migrate/20150730052407_add_missing_indexes.spree.rb

```diff
@@ -0,0 +1,26 @@
+# This migration comes from spree (originally 20150314013438)
+class AddMissingIndexes < ActiveRecord::Migration
+  def change
+    add_index   :spree_promotion_rules_users,
+                        [:user_id, :promotion_rule_id],
```

Comment:
__[rubocop]__
* Align the parameters of a method call if they span more than one line.